service/src/main/java/bio/terra/policy/service/pao/PaoService.java

```diff
@@ -256,21 +256,18 @@ public PolicyUpdateResult updatePao(
     // Now integrate the adds into the attribute set
     for (PolicyInput addPolicy : addAttributes.getInputs().values()) {
       PolicyInput existingPolicy = targetPao.getAttributes().lookupPolicy(addPolicy);
```

Comment:
`targetPao.getAttributes` is the same as `attributesToUpdate`
We are mutating the targetPao in place. Is that OK?
Maybe add a comment around line 240 to note that it is not a copy.